recipes/obspy/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+## echo "print('Testing an example package.')" > $SP_DIR/example.py
+cp -r $RECIPE_DIR/.. $SRC_DIR
```

Comment:
No idea, was wondering too and forgot to double-check. Seems it's an artifact of old tries at getting the build done.. I'll remove it.